torch/optim/rprop.py

```diff
@@ -65,13 +65,12 @@ def _init_group(self, group, params, grads, prevs, step_sizes):
                     # Complex Number should be as if they are two independent real numbers.
                     # Hence the step_size shouldn't be zero for imaginary part.
                     state["step_size"] = (
-                        grad.new()
-                        .resize_as_(grad)
+                        torch.empty_like(grad)
                         .fill_(complex(group["lr"], group["lr"]))
                     )
                 else:
                     state["step_size"] = (
-                        grad.new().resize_as_(grad).fill_(group["lr"])
+                        torch.empty_like(grad).fill_(group["lr"])
```

Comment:
could this instead be `torch.full_like(grad, group["lr"])`?

Reply:
@vadimkantorov Thanks for the suggestion! 

@janeyx99, do you know the answer to @vadimkantorov 's 2nd question?

Reply:
Currently, it is the same shape as param/grad